Week 2 - Arrays/scrabble/scrabble.c

```diff
@@ -0,0 +1,72 @@
+#include <cs50.h>
+#include <ctype.h>
+#include <stdio.h>
+
+int getIndex(char key);
+int getValue(char key);
+int countPoints(string word);
+
+int SIZE = 26;
+char KEYS[26] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
+                 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'};
+int VALUES[26] = {1, 3, 3, 2, 1, 4, 2, 4, 1, 8, 5, 1, 3, 1, 1, 3, 10, 1, 1, 1, 1, 4, 4, 8, 4, 10};
+
+int main(void)
+{
+    string p1 = get_string("Player1: ");
```

Comment:
We usually prefer more meaningful names for our variable to clearly see what was the idea behind
```suggestion
    string player1 = get_string("Player1: ");
```